server/run-dev.sh

```diff
@@ -2,4 +2,6 @@
 
 ./migration-run.sh
 
+export INTERNAL_ACCESS_KEY=anything
```

Comment:
Will this overwrite a value set in `.env` at the project root? If so, I'd probably prefer to remove it to avoid confusion. I normally just have `INTERNAL_ACCESS_KEY` set in my `.env` file.

The current setup is a bit awkward, since using `server/run-dev.sh` won't load those environment variables but `run-dev-server.sh` will. Maybe we should just move the `.env` file to `server/.env` since I think we're only really messing with server values anyways. If we need to tweak anything on the client we could add one at `client/.env` eventually.

Reply:
fixed